app/screens/login.tsx

```diff
@@ -24,17 +31,17 @@ export const LoginScreen: React.FunctionComponent<LoginProps> = observer(props =
 	  console.log('login success');
       navigationStore.dispatch(NavigationActions.navigate({ routeName: 'packagesTabList' }))
     } else {
-	  console.log(`login failed: ${JSON.stringify(loginReq)}`);
-	  displayLoginError();
+		let errorMsg = undefined;
+		if(loginReq.status === 401 ) {
+			errorMsg = 'The username or password is incorrect';
+		} else {
+			errorMsg = 'An error occurred when logging in please try again later';
+		}
+		//console.log(`login failed: ${JSON.stringify(loginReq)}`);
+		toggleAlert(/*errorMsg*/);
     }
```

Comment:
I think its best that this logic should be inside the network request function. 
we want to be able to catch each network error

Reply:
please be specific, which logic i can't see the updated file here looks to be outdated.
I suppose you speak about 
if(loginResponse.status === 401 ) {
				errorMsg = 'The username or password is incorrect';
			} else {
				errorMsg = 'An error occurred when logging, please try again later';
			}

to move into the login function itself?

Reply:
will be done in next PR, i will cut the modal logic from the component